tests/unit/dev_support.py

```diff
@@ -54,13 +54,17 @@ def _wet(func):
 
 
 # Actual test cases:
-def test_dry_function(): _dry(_function)
+def test_dry_function():
+    _dry(_function)
```

Comment:
Probably @meng-sun / @gfaline changed this while trying to fix pep8 errors. That's my best guess.